README.md

```diff
@@ -54,9 +54,10 @@ Original Developers
  * Mads Ynddal - [baekalfen](https://github.com/Baekalfen)
  * Troels Ynddal - [troelsy](https://github.com/troelsy)
 
-GitHub Collaborators
---------------------
+Core Developers
+---------------
 
+ * Mads Ynddal - [baekalfen](https://github.com/Baekalfen)
  * Kristian Sims - [krs013](https://github.com/krs013)
```

Comment:
I removed your name here, as I can see that it was ambiguous.

We'll add a different list for contributors in v1.1.0